sql/2021/privacy/most_common_featurepolicy_permissionspolicy_directive_values.sql

```diff
@@ -0,0 +1,141 @@
+#standardSQL
+# Most common directives for Feature-Policy or Permissions-Policy
+
+WITH page_ranks AS (
+  SELECT
+    client,
+    page,
+    rank
+  FROM
+    `httparchive.almanac.summary_response_bodies`
+  WHERE
+    date = '2021-07-01' AND firstHtml = TRUE
```

Comment:
Formatting nit, list WHERE clauses separately where it makes no difference:

```suggestion
    date = '2021-07-01' AND
    firstHtml = TRUE
```

(applies throughout)

Reply:
Fixed in e878bce